frontend/app/_components/client-dashboard-components/FormCreateProject.js

```diff
@@ -0,0 +1,238 @@
+            value={formData.title}
+            onChange={handleChange}
+            className="px-5 py-3 bg-primary-200 text-primary-800 w-full shadow-sm rounded-sm"
+            required
+          />
+        </div>
+        <div className="space-y-2">
+          <label
+            htmlFor="budget"
+            className="block text-sm md:text-base font-medium"
+          >
+            How much is your budget?
+          </label>
+          <input
+            name="budget"
+            value={formData.budget}
+            onChange={handleChange}
+            type="number"
+            className="px-5 py-3 bg-primary-200 text-primary-800 w-full shadow-sm rounded-sm"
+            required
+          />
+        </div>
+
+        <div className="space-y-2">
+          <label
+            htmlFor="startDate"
+            className="block text-sm md:text-base font-medium"
+          >
+            When do you want to start?
+          </label>
+          <input
+            name="startDate"
+            value={formData.startDate}
+            onChange={handleChange}
+            type="date"
+            className="px-5 py-3 bg-primary-200 text-primary-800 w-full shadow-sm rounded-sm"
+            required
+          />
+        </div>
+
+        <div className="space-y-2">
+          <label
+            htmlFor="startDate"
+            className="block text-sm md:text-base font-medium"
+          >
+            When do you want finish this project?
+          </label>
+          <input
+            name="endDate"
+            value={formData.endDate}
+            onChange={handleChange}
+            type="date"
+            className="px-5 py-3 bg-primary-200 text-primary-800 w-full shadow-sm rounded-sm"
+            required
+          />
+        </div>
+
+        <div className="space-y-2">
+          <label
+            htmlFor="deadline"
+            className="block text-sm md:text-base font-medium"
+          >
+            Deadline
+          </label>
+          <input
+            name="deadline"
+            value={formData.deadline}
+            onChange={handleChange}
+            type="date"
+            className="px-5 py-3 bg-primary-200 text-primary-800 w-full shadow-sm rounded-sm"
+            required
+          />
+        </div>
+        <div className="space-y-2">
+          <label
+            htmlFor="status"
+            className="block text-sm md:text-base font-medium"
+          >
+            Select your project status
+          </label>
+          <select
+            name="status"
+            value={formData.status}
+            onChange={handleChange}
+            className="px-5 py-3 bg-primary-200 text-primary-800 w-full shadow-sm rounded-sm"
+            required
+          >
+            <option value="">Choose Status</option>
+            <option value="Pending">Pending</option>
+            <option value="In Progress">In Progress</option>
+            <option value="Completed">Completed</option>
```

Comment:
I tried to submit a new project using your form, but the request id failed. I assumed it is because you are using the wrong status values here. It should be : "pending", "in-progress", "completed". It's how it designed in backend, and it's case-senstive.
And please try it before you create the Pull Request.
![image](https://github.com/user-attachments/assets/ce7b5b00-3148-4dea-8189-de085d75ad63)

Reply:
the reson was inprogress that diff from api , no it works